tabular/src/autogluon/tabular/task/tabular_prediction/predictor.py

```diff
@@ -90,9 +91,59 @@ def __init__(self, learner):
         self.eval_metric = self._learner.eval_metric
         self.label_column = self._learner.label
         self.feature_metadata = self._trainer.feature_metadata
-        self.class_labels = self._learner.class_labels
-        self.class_labels_internal = self._learner.label_cleaner.ordered_class_labels_transformed
-        self.class_labels_internal_map = self._learner.label_cleaner.inv_map
+
+    @property
+    def class_labels(self):
+        return self._learner.class_labels
```

Comment:
What will be the behavior of `class_labels` if it's a regression task?

Reply:
It will return None